src/aind_metadata_mapper/U19/u19_etl.py

```diff
@@ -0,0 +1,405 @@
+"""Sets up the U19 ingest ETL"""
```

Comment:
change the name of this module to procedures.py (to match naming convention of the rest of repo).